src/classes/ ffTestStud.cls

```diff
@@ -0,0 +1,194 @@
+public with sharing class ffTestStud {
+
+  // Constructor.
+Public AccountDeDuper () {
+
+}
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
+public void BADSTUFF__4() {}
```

Comment:
* This method is empty, I suggest to remove it.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.